exo/main.py

```diff
@@ -221,16 +267,36 @@ def handle_exit():
       print("Error: Model name is required when using 'run' command or --run-model")
       return
     await run_model_cli(node, inference_engine, model_name, args.prompt)
+  elif args.command == "eval" or args.command == 'train':
+    data_path = args.data
+    train, val, test = load_dataset(data_path)
+    model_name = args.model_name
+    if args.command == 'eval':
+      if not model_name:
+        print("Error: Much like a human, I can't evaluate anything without a model")
```

Comment:
I do love this line, but maybe add in something like `Please pass in a parameter for model_name (e.g. "exo eval llama-3.2-3b")`  so that users don't have to parse through the code in main.py to get this working (replace with a working example command if I didn't get that right)

Reply:
I appreciate the sentiment of writing good error messages but this is still very much a work in progress